src/GovUk.Education.ExploreEducationStatistics.Admin.Tests/Services/UserManagementServiceTests.cs

```diff
@@ -203,14 +206,305 @@ await using (var userAndRolesDbContext = InMemoryUserAndRolesDbContext(userAndRo
             VerifyAllMocks(userRoleService);
         }
 
+        [Fact]
+        public async Task InviteUser()
+        {
+            var role = new IdentityRole
+            {
+                Id = Guid.NewGuid().ToString(),
+                Name = "Role 1",
+                NormalizedName = "ROLE 1"
+            };
+
+            var usersAndRolesDbContextId = Guid.NewGuid().ToString();
+            await using (var usersAndRolesDbContext = InMemoryUserAndRolesDbContext(usersAndRolesDbContextId))
+            {
+                await usersAndRolesDbContext.AddRangeAsync(role);
+                await usersAndRolesDbContext.SaveChangesAsync();
+            }
+
+            var release = new Release();
+
+            var contentDbContextId = Guid.NewGuid().ToString();
+            await using (var contentDbContext = InMemoryApplicationDbContext(contentDbContextId))
+            {
+                await contentDbContext.AddRangeAsync(new Release());
+                await contentDbContext.SaveChangesAsync();
+            }
+
+            var emailTemplateService = new Mock<IEmailTemplateService>(Strict);
+
+            emailTemplateService.Setup(mock =>
+                    mock.SendInviteEmail("test@test.com"))
+                .Returns(Unit.Instance);
+
+            await using (var userAndRolesDbContext = InMemoryUserAndRolesDbContext(usersAndRolesDbContextId))
+            await using (var contentDbContext = InMemoryApplicationDbContext(contentDbContextId))
+            {
+                var service = SetupUserManagementService(
+                    contentDbContext: contentDbContext,
+                    usersAndRolesDbContext: userAndRolesDbContext,
+                    emailTemplateService: emailTemplateService.Object);
+
+                var result = await service.InviteUser("test@test.com",
+                    role.Id,
+                    new List<AddUserReleaseRoleViewModel>
+                    {
+                        new ()
+                        {
+                            ReleaseId = release.Id,
+                            ReleaseRole = Approver,
+                        },
+                    });
+
+                result.AssertRight();
+            }
+
+            await using (var usersAndRolesDbContext = InMemoryUserAndRolesDbContext(usersAndRolesDbContextId))
+            {
+                var userInvites = usersAndRolesDbContext.UserInvites
+                    .Where(invite => invite.Email == "test@test.com")
```

Comment:
If we have this `Where`, we don't really prove that only exactly one `UserInvite` was created. Perhaps there's a bug that creates an additional, invalid invite for example.

Would suggest removing the `Where` so that we can rest assured that only is created. We could do a similar thing across the rest of the test cases and checks against `UserReleaseInvites`.